data/faq/steal_project_idea.md

```diff
@@ -0,0 +1 @@
+The whole point of 'Open Source Software' is to encourage "stealing". If you do not honor original authors, licenses, and the "creator" of an original idea, there is no shame in saying "he had a great idea, I think I can implement it". On the plus side, we will always take into account who came up with something and if you help others succeed, we'll definitely credit you for that. 
```

Comment:
This is not correct.
If you do not honor them (by not saying they had the idea first) that is wrong.
If you follow license rules, disclaim original idea creators and authors you probably are fine. Usually talking to the involved parties is another good idea.

Could you limit the lines to 80 chars?